src/vs/workbench/contrib/timeline/browser/timelinePane.ts

```diff
@@ -1159,7 +1159,14 @@ class TimelineTreeRenderer implements ITreeRenderer<TreeElement, FuzzyScore, Tim
 		@IThemeService private themeService: IThemeService,
 	) {
 		this.actionViewItemProvider = createActionViewItem.bind(undefined, this.instantiationService);
-		this._hoverDelegate = getDefaultHoverDelegate('mouse');
+		this._hoverDelegate = this.instantiationService.createInstance(WorkbenchHoverDelegate, 'element', false, {
```

Comment:
```suggestion
		this._hoverDelegate = this.instantiationService.createInstance(WorkbenchHoverDelegate, 'element', true, {
```

Reply:
I think we should use instantHover if we use `element` positioning

Reply:
SCM doesn't use it right now, originally I think instant hover was only going to be for things like views where they're only represented by icons, though it doesn't seem to do that for panel titles 🤔

Reply:
Actually, scm does use it

Reply:
It's not clear what `instantHover` actually does, and apparently it's `timeLimit` is entirely controlled by notebooks:

![image](https://github.com/user-attachments/assets/a5aa8633-a035-4ba0-a969-4051451ae427)

Reply:
Also it's 200ms by default, that's not "instant" either 😕 

https://github.com/microsoft/vscode/blob/09ba66b21b65ced16b17f6c5ed8bc2b0a9cbcd16/src/vs/platform/hover/browser/hover.ts#L23

Reply:
Maybe this needs a better name. Instant hover means, it should show instantly if there was a hover shown from the same delegate less than `timeLimit`ms ago